log/lvl.go

```diff
@@ -0,0 +1,385 @@
+// Package log is an output-library that can print nicely formatted
```

Comment:
How come all of this appear as new files ? and not just renaming + some adjustements ? Is it because of IntelliJ ?
`git mv` should not do that.